beetsplug/edit.py

```diff
@@ -313,8 +322,8 @@ def apply_data(self, objs, old_data, new_data):
             if forbidden:
                 continue
 
-            id = int(old_dict['id'])
-            apply(obj_by_id[id], new_dict)
+            id_ = int(old_dict['id'])
+            apply_(obj_by_id[id_], new_dict)
```

Comment:
These were renamed in order not to have overly-zealous IDEs complain about overriding a built-in symbol.